src/v2/Components/NavBar/Menus/MobileNavMenu/__tests__/MobileNavMenu.jest.tsx

```diff
@@ -74,15 +72,17 @@ describe("MobileNavMenu", () => {
       expect(linkText).toContain("Artists")
       expect(linkText).not.toContain("Career Stages")
 
-      const simpleLinks = linkContainer.children(MobileLink)
-      expect(simpleLinks.length).toBe(8)
-      ;(MENU_DATA.links as SimpleLinkData[])
-        .slice(2)
-        .map(({ href, text }, index) => {
-          const simpleLink = simpleLinks.at(index)
-          expect(href).toEqual(simpleLink.prop("href"))
-          expect(text).toEqual(simpleLink.text())
-        })
+      // TODO: Not sure these specs make any kind of sense:
+      // We don't use the data for rendering these links (why?)
+      // const simpleLinks = linkContainer.children(MobileLink)
+      // expect(simpleLinks.length).toBe(8)
+      // ;(MENU_DATA.links as SimpleLinkData[])
+      //   .slice(2)
+      //   .map(({ href, text }, index) => {
+      //     const simpleLink = simpleLinks.at(index)
+      //     expect(href).toEqual(simpleLink.prop("href"))
+      //     expect(text).toEqual(simpleLink.text())
+      //   })
```

Comment:
We have this `menuData.ts` which theoretically contains the data required to render the nav, but we only use some of it and disregard other parts of it. But it's used in specs in some places like this. Previously some of it was being inserted as props when the data was constant.

I'm just trying to understand some of the "why" for this so that I can try to push this in one direction or the other. I see no reason why the entire menu data can't be expressed as configuration shared between the Desktop and Mobile versions?

cc @damassi @mzikherman

Reply:
I have no clue! That seems reasonable to me.

Reply:
@xtina-starr was mostly responsible for the large drop-down portions of the nav, but as far as I can remember, when it was first released, everything was driven by `menuData` as one would expect. But I only served as a drive-by PR reviewer so am not 100% sure; Christina would know more I bet.